e2e/verify.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
I usually recommend that we use JavaScript and Node for these sorts of things since it's often the case that new contributors trip over shell scripts when they're using Windows.

suggestion(non-blocking): I think we should consider doing the assertions in JavaScript

Reply:
Hmm I hadn't thought of windows. I didn't use javascript because it was fewer lines and easier to code in bash since `jq` is quite powerful. In JS we have to split by lines and parse each line then search for the line we want.

I can update it to use JS though for the sake of portability.